### ISSUE TITLE
Questions about the items in the prompt dataset

### ISSUE THREAD

**Issue by JunyiChE** [jwzhanggy/Graph_Toolformer#9]
Hi,
Very impressive and interesting work.

However, I have one question about the prompt dataset. The number of prompts in the dataset uploaded seems much less than that claimed in the paper.

For example, in the table 3 of the paper, it is said Amazon prompt dataset contains 2,252,890 prompts. However, in the repository, I can only find 1600 items in the training set and 160 items in the test set. Is there anything I missed or misunderstood?

Thanks for your clarification and looking forward to your kind reply!

**Comment by JunyiChE** [comment]
![image](https://github.com/jwzhanggy/Graph_Toolformer/assets/151556247/1c6cdb38-81db-4433-bb36-01168d401230)
Table that describes the prompts number in the paper
